manim/mobject/coordinate_systems.py

```diff
@@ -765,6 +765,19 @@ def construct(self):
                     f"x={x} not located in the range of the graph ([{self.p2c(graph.get_start())[0]}, {self.p2c(graph.get_end())[0]}])",
                 )
 
+    def input_to_graph_coords(self, x: float, graph: "ParametricFunction") -> Tuple:
+        """
+        Returns a tuple of the axis relative coordinates of the point
+        on the graph based on the x-value given.
+        """
+        return x, graph.underlying_function(x)
```

Comment:
Could you add an example (doesn't need to be visual)  + more docs for the parameters of this method?, it's a little confusing how this is functionally different from `i2gp`. Also, I'm a little unsure of adding a public method which doesn't really serve a unique purpose 🤔

Reply:
> I'm a little unsure of adding a public method which doesn't really serve a unique purpose

I sort of see what you mean, but I also do see myself using it, like passing this method into `Tex` or `Text` to label a coordinate.